packages/deployer/src/cli/commands.ts

```diff
@@ -78,6 +78,6 @@ export async function removeWithReceipt(receiptFilename: string) {
     await remove(airnodeIdShort, stage, cloudProvider, region);
   } catch (err) {
     logger.warn(`Failed removing configuration, skipping`);
-    logger.warn(err.toString());
+    logger.warn((err as Error).toString());
```

Comment:
It's better to just say `(err: Error)` directly in the catch clause.

Reply:
That was my first thought too, but that produces the following error:

![catch-error-type](https://user-images.githubusercontent.com/4114009/131850904-5bc33a07-a238-443c-a968-d726d9afc7c4.png)

Reply:
I see. 

This is a bit annoying pattern but I don't mind this too much.